### PR TITLE
fix: attempt to fix temp chart issues

### PR DESCRIPTION
**Pull request by pedrolamas**
Related with #453

Signed-off-by: Pedro Lamas <pedrolamas@gmail.com>

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -33,11 +33,10 @@ import { FiltersPlugin } from './plugins/filters'
 import { SocketPlugin } from './plugins/socketClient'
 import { ColorSetPlugin } from './plugins/colorSet'
 import { DayJSPlugin } from './plugins/dayjs'
-import { plugin } from 'echarts-for-vue'
 
 // Import ECharts
-// import * as echarts from 'echarts'
-import * as echarts from 'echarts/core'
+import ECharts from 'vue-echarts'
+import { use } from 'echarts/core'
```
Comment (matmen) [inline src/main.ts:39]:
```suggestion
import { use as echartsComponents } from 'echarts/core'
```

File: src/main.ts [hunk 2]
```diff
@@ -70,7 +68,7 @@ Vue.component('vue-headful', vueHeadful)
 // Use any Plugins
 
 // Configure echarts
-echarts.use([
+use([
```
Comment (matmen) [inline src/main.ts:71]:
```suggestion
echartsComponents([
```

**Review by matmen — COMMENTED** [review]
LGTM, but maybe change these so the code is a little more descriptive?

File: src/main.ts (hunk 1 above)
Comment (pedrolamas) [reply src/main.ts]: renamed to `echartsUse`

**Review by matmen — APPROVED** [review]

**Pull request merged**